package.json

```diff
@@ -4,6 +4,10 @@
   "description": "Declarative HTTP requests with React.",
   "main": "lib/index.js",
   "module": "es/index.js",
+  "engine-strict": true,
+  "engines": {
+    "node": ">= 8"
+  },
```

Comment:
Thanks for adding this! Would you mind moving it to a separate commit?

Reply:
On second thought, I don't think we should add this. Node 8 is required for _building_ the lib, but not for consuming it. And the engines field is about consumption.